keyboards/1upkeyboards/1upsuper16v3/info.json

```
@@ -0,0 +1,195 @@
+            "solid_reactive_simple": true,
+	        "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "WS2812",
+        "layout": [
+
+        { "flags": 2, "x": 99, "y": 1 },
+        { "flags": 2, "x": 70, "y": 1 },
+        { "flags": 2, "x": 42, "y": 1 },
+        { "flags": 2, "x": 14, "y": 1 },
+        { "flags": 2, "x": 4, "y": 4 },
+        { "flags": 2, "x": 4, "y": 12 },
+
+        { "flags": 4, "matrix": [0, 0], "x": 28, "y": 8 },
+        { "flags": 4, "matrix": [0, 1], "x": 84, "y": 8 },
+
+        { "flags": 2, "x": 4, "y": 20 },
+        { "flags": 2, "x": 4, "y": 28 },
+
+        { "flags": 4, "matrix": [0, 4], "x": 28, "y": 24 },
+        { "flags": 4, "matrix": [0, 5], "x": 84, "y": 24 },
+
+        { "flags": 2, "x": 4, "y": 36 },
+        { "flags": 2, "x": 4, "y": 44 },
+
+        { "flags": 4, "matrix": [0, 8], "x": 28, "y": 40 },
+        { "flags": 4, "matrix": [0, 9], "x": 84, "y": 40 },
+
+        { "flags": 2, "x": 4, "y": 52 },
+        { "flags": 2, "x": 4, "y": 60 },
+        { "flags": 2, "x": 14, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 12], "x": 28, "y": 56 },
+
+        { "flags": 2, "x": 42, "y": 63 },
+        { "flags": 2, "x": 70, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 13], "x": 84, "y": 56 },
+
+        { "flags": 2, "x": 99, "y": 63 },
+        { "flags": 2, "x": 126, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 14], "x": 140, "y": 56 },
+
+        { "flags": 2, "x": 154, "y": 63 },
+        { "flags": 2, "x": 182, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 15], "x": 196, "y": 56 },
+
+        { "flags": 2, "x": 210, "y": 63 },
+        { "flags": 2, "x": 220, "y": 60 },
+        { "flags": 2, "x": 220, "y": 52 },
+        { "flags": 2, "x": 220, "y": 44 },
+
+        { "flags": 4, "matrix": [0, 11], "x": 196, "y": 40 },
+        { "flags": 4, "matrix": [0, 10], "x": 140, "y": 40 },
+
+        { "flags": 2, "x": 220, "y": 36 },
+        { "flags": 2, "x": 220, "y": 28 },
+
+        { "flags": 4, "matrix": [0, 7], "x": 196, "y": 24 },
+        { "flags": 4, "matrix": [0, 6], "x": 140, "y": 24 },
+
+        { "flags": 2, "x": 220, "y": 20 },
+        { "flags": 2, "x": 220, "y": 12 },
+
+        { "flags": 4, "matrix": [0, 3], "x": 196, "y": 8 },
+        { "flags": 4, "matrix": [0, 2], "x": 140, "y": 8 },
+
+        { "flags": 2, "x": 220, "y": 4 },
+        { "flags": 2, "x": 210, "y": 1 },
+        { "flags": 2, "x": 182, "y": 1 },
+        { "flags": 2, "x": 154, "y": 1 },
+        { "flags": 2, "x": 126, "y": 1 }
+
+        ]
+    },
```

matching community layout(s) identified
```suggestion
    },
    "community_layouts": [ "ortho_4x4" ],
```